app/src/components/market/sections/market_buy/market_buy.tsx

```diff
@@ -38,6 +30,14 @@ import { TransactionDetailsRow, ValueStates } from '../../common/transaction_det
 import { ViewCard } from '../../common/view_card'
 import { WalletBalance } from '../../common/wallet_balance'
 
+import { stripIndents } from 'common-tags'
```

Comment:
Moving the imports this way makes the linter fail. Maybe your IDE isn't reading the eslint config?